spec/frequency_classes_service_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'frequency_classes_service'
+
+describe 'FrequencyClassesService' do
+  let(:frequency_classes_service) { FrequencyClassesService.new }
+  let(:sample_word) { 'haben' }
+
+  describe '#initialize' do
+    it 'should return a FrequencyClassesService instance' do
+      expect(frequency_classes_service).to be_instance_of(
+        FrequencyClassesService
+      )
+    end
+  end
+
+  describe '#calculate' do
+    it 'should calculate the frequency class for haben to match 3' do
```

Comment:
this looks good, can you provide some more specs like it to provide a more complete description of what to expect as a user of the object?